geoserver/webapp/src/docker/Dockerfile

```diff
@@ -1,8 +1,17 @@
-FROM georchestra/base
+FROM jetty:9.3-jre8
+
+
+RUN apt-get update && \
+    apt-get install -y unzip && \
+    rm -rf /var/lib/apt/lists/*
 
 ADD . /
 ADD https://github.com/georchestra/geoserver_minimal_datadir/archive/master.zip /tmp/datadir.zip
 
-RUN chmod +x /root/start.sh
+VOLUME [ "/var/local/geoserver", "/var/local/geodata" ]
+
+ENTRYPOINT [ "/docker-entrypoint.sh" ]
+CMD [ "java","-Djava.io.tmpdir=/tmp/jetty", "-Dgeorchestra.datadir=/etc/georchestra",                     \
+      "-DGEOSERVER_DATA_DIR=/var/local/geoserver/", "-Dgeofence.dir=/etc/georchestra/geoserver/geofence", \
```

Comment:
missing volume and option here : GEOWEBCACHE_CACHE_DIR (`/var/local/geowebcache`)